jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -414,7 +414,7 @@
             * [Internal Jenkins Results](${{JOB_URL}}/${{BUILD_NUMBER}})
 
             Please reference the [list of currently known flakes](https://github.com/kubernetes/kubernetes/issues?q=is:issue+label:kind/flake+is:open) when examining this failure. If you request a re-test, you must reference the issue describing the flake.
-          trigger-phrase: 'e2e\s+test'
+          trigger-phrase: 'gce e2e test'
```

Comment:
commenting "federation gce e2e test" will also trigger this.

and I predict a lot of frustrated developers wondering why "@k8s-bot e2e test this" isn't working. we probably need to widely announce and educate.